packages/ui/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@webpack-cli/ui",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www"
+  },
+  "dependencies": {
+    "cookie-parser": "~1.4.3",
```

Comment:
Ideally we could add `body-parser`. For the next step :)

Reply:
You mean add it as dependency 🤔